src/elements/list-box.svelte

```diff
@@ -0,0 +1,159 @@
+<svelte:options immutable tag='v-list-box' />
+
+<script lang='ts'>
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+// disabled to pass untyped object to cx library in template
+
+import cx from 'classnames';
+import { afterUpdate } from 'svelte';
+
+import { dispatcher } from '../lib/dispatch';
+import { addStyles } from '../lib/index';
+import { htmlToBoolean } from '../lib/boolean';
+
+export let disabled: string;
```

Comment:
shouldn't we set the default of this to be false?

Reply:
Since `isDisabled` is the boolean converted value I'm leaving this defaulted to undefined to match how users would use it - they either have the disabled attribute on the component which will accept the value as a string or it's not there at all and therefore undefined.